tests/runtime/uprobe

```diff
@@ -52,6 +52,16 @@ EXPECT Attaching 1 probe...
 TIMEOUT 5
 BEFORE ./testprogs/mountns_wrapper uprobe_test
 
+# Note that we don't expect the exact function1 name because CI runs in a docker
```

Comment:
Does this mean that when this is not running in a container, this test will fail?

Reply:
No, it will pass if the regex is left as is, because if symbol resolution works, as it would when not running in a container, then the output would be function1, which ```[0-9a-fx]*``` matches.

Reply:
Well, technically it does, but only b/c it's not matching the entire line and a single letter is sufficient. I'd prefer something more robust like `^(0x[0-9a-f]+|function1)$` (you may need to escape some characters).